### DISCUSSION TITLE
Could you add the bale collecting function to the KUHN SW 4014?

### DISCUSSION THREAD

**Discussion by rento112** [Courseplay/courseplay#6862] [Q&A] [answered]
With the new release of v6.03.00043. Could you give the bale course function for the bale wrapper KUHN  SW 4014?

Edit, reading the manual, but combine confuses me, i dont think any combine has the function for bales... except for chaff. Corn chaff cant be turned into bales it  seems when i tried it.
I think I'm misunderstand this function. I'm still new to FS19

**Comment by Tensuko** [comment]
It is for collecting bales and drive them to an unload place.
For the bale wrapper we would need some changes, as we don't drive them away, but I like the Idea.

What manual do you refer to ?
That mode got nothing to do with combines anymore.
If the first text I did on the website is confusing, let me know what part it is and I will change that (I guess the picture must be changed too).

**Reply by rento112** [reply]
I tried to have my tractor with arcusin autostacker, but it wouldnt do anything. I set it to bale collecting on a spcific  field. Wouldnt work on the mod by giants, the 8 bale round autostacker, Ursus, but that didnt do anything too.

**Reply by Tensuko** [reply]
Pls have a look at #6859

**Reply by rento112** [reply]
> 
> 
> Pls have a look at #6859

Yeah, i had it had setup like that, it didntt work.

**Reply by rento112** [reply]
Alright. tried again, but it is starting to work..... but it keeps stopping after 1 bale or 2
By the waay, Maybe also implement thhe ability to filter bales, so that it choose to only go for wrapped or not wrapped bales.

Edit: It would also seem that the latest update broke thhe lane offset. It normally makes my autostacker go left a bit without touching it, but not it doesnt have it automatically anymore

**Reply by Tensuko** [reply]
Reset your vehicle, I just tried it and I didn't need any offset.

**Comment by pvaiko** [comment, marked as answer]
https://github.com/Courseplay/courseplay/releases/tag/6.03.00050 adds support for bale wrappers

**Reply by rento112** [reply]
> 
> 
> https://github.com/Courseplay/courseplay/releases/tag/6.03.00050 adds support for bale wrappers

Hey. here's another idea that would be awesome, maybe add the feature for bale collectors to filter what bales they collect, like wrapped vs unwrapped, so that we can set a bale wrapper too go for bales while a bale collector can collect only wrapped bales at the same time.

Oh and thanks for the  featurre!

**Reply by Tensuko** [reply]
Sure that is a nice Idea, but I already see the incomeing Issues when wrapper and collector snuggling with the baler itself.
Yes we have an avoid system, but not sure how well this will work with towed tools that have an offset.
